config/_default/menus/menus.en.yaml

```diff
@@ -1,7665 +0,0 @@
-main_left:
```

Comment:
I think we just need to remove `main_left` `main_right` and any `foot_*` menus from the menu files. We want to keep `main` and `api` still in here as they are specific to docs.